CustomSundials/default.nix

```diff
@@ -7,7 +7,10 @@
 , liblapack
 , gfortran
 , suitesparse
-, lapackSupport ? true }:
+, lapackSupport ? true
+, indexType ? "int32_t"
```

Comment:
Please document theses values. What is the impact, why did you choose these default?

Reply:
Updated and left the default values up to sundials.